examples/08_join_aggregation.py

```diff
@@ -1,302 +1,311 @@
 """
-Self-aggregation on MovieLens
-=============================
+AggJoiner on a credit fraud dataset
+===================================
 
-MovieLens is a famous movie dataset used for both explicit
-and implicit recommender systems. It provides a main table,
-"ratings", that can be viewed as logs or transactions, comprised
-of only 4 columns: ``userId``, ``movieId``, ``rating`` and ``timestamp``.
-MovieLens also gives a contextual table "movies", including
-``movieId``, ``title`` and ``types``, to enable content-based feature extraction.
+Many problems involve tables whose entities have a one-to-many relationship.
+To simplify aggregate-then-join operations for machine learning, we can include
+the |AggJoiner| in our pipeline.
 
-From the perspective of machine-learning pipelines, one challenge is to
-transform the transaction log into features that can be fed to supervised learning.
+In this example, we are tackling a fraudulent loan detection use case.
+Because fraud is rare, this dataset is extremely imbalanced, with a prevalence of around
+1.4%.
 
-In this notebook, we only deal with the main table "ratings".
-Our objective is **not to achieve state-of-the-art performance** on
-the explicit regression task, but rather to illustrate how to perform
-feature engineering in a simple way using |AggJoiner| and |AggTarget|.
-Note that our performance is higher than the baseline of using the mean
-rating per movies.
+The data consists of two distinct entities: e-commerce "baskets", and "products".
+Baskets can be tagged fraudulent (1) or not (0), and are essentially a list of products
+of variable size. Each basket is linked to many products, e.g. basket 1 can have
```

Comment:
```suggestion
of variable size. Each basket can be linked to many products, e.g. basket 1 can have
```

Reply:
"is linked to at least one" if we want to be thorough